tests/console/system_prepare.pm

```diff
@@ -91,6 +92,12 @@ sub run {
         }
     }
 
+    # enable multiversion for kernel-default based on bsc#1097111, for migration continuous cases only
+    if (get_var('FLAVOR', '') =~ /Continuous-Migration/) {
+        record_soft_failure 'workaround for bsc#1097111, enable multiversion for kernel after migration';
```

Comment:
you can move record_soft_failure into the disable_kernel_multiversion_temporarily